### DISCUSSION TITLE
[RFC] Transfer this category to stac-utils discussions

### DISCUSSION THREAD

**Discussion by gadomski** [radiantearth/stac-spec#1218] [STAC Software]
- Start date: 2023-03-14

# Summary

We will enable discussions on the [stac-utils organization](https://github.com/stac-utils) and transfer discussions in the "PySTAC" category of the **stac-spec** discussions there.

# Motivation

**PySTAC** doesn't fully capture the scope of this discussion category -- we've used this to talk about [stactools](https://github.com/stac-utils/stactools), [stactools-packages](https://github.com/stactools-packages/), and more. I'd like to create some more documentation around the process for including a repository in **stac-utils**, standardizing tooling for Python **stac-utils** repos, and more, and it is starting to feel a little strange to keep creating RFCs here.

# Drawbacks

We're creating more places where people have to look for information.

# Rationale and alternatives

- Organizations have their own discussion boards, and (to me) it makes sense to have discussion about **stac-utils** in the **stac-utils** organization.
- We could rename this category and keep using the **stac-spec** discussion area, which would at least be better than the current setup.
- If we don't do anything, (IMO) it's confusing to find writups about **stactools** in a category called "PySTAC"
- We'll link back to these **stac-spec** discussions from the **stac-utils** discussions and README.

**Comment by m-mohr** [comment]
No strong feeling, but having a central place seems nice as I have issues keeping track of this discussion board already so would likely miss even more if we split up. I'm wondering whether renaming the category would make sense? We could also add more... Python, R, JS, Rust, C#, Go, ...

**Reply by gadomski** [reply]
> I'm wondering whether renaming the category would make sense? We could also add more... Python, R, JS, Rust, C#, Go, ...

Yeah, I'd be fine with that as well -- to me, it feels more natural to have spec-related discussions here, and code-related discussions in **stac-utils**, but the decrease in visibility is a downside for sure. If we rename this category that would be Good Enough™ for now, IMO, as there aren't really that many discussions to warrant a breakout into language-specific categories (yet).

**Reply by gadomski** [reply]
@m-mohr if you have permissions, can you update this category to just be "STAC Software" or something similar? That should be fine for now.

**Reply by m-mohr** [reply]
@gadomski Done. Also changed from Q&A for Open-ended discussion style as it seems we don't really have Q&A content in here right now.

**Reply by gadomski** [reply]
Thanks!
